reference-guide/routes/override-a-route.md

```diff
@@ -509,8 +509,7 @@ if ForestLiana::UserSpace.const_defined?('UserController')
 
     # Create a User
     def create
-      checker = ForestLiana::PermissionsChecker.new(@resource, 'addEnabled', @rendering_id, user_id: forest_user['id'])
-      return head :forbidden unless checker.is_authorized?
+	  ForestLiana::Ability::forest_authorize!('add', forest_user, @resource)
```

Comment:
The rendering is automatically fetched from the request now ? Just to confirm 😇

Reply:
I can add in the snippet a begin … rescue for return a HTTP response, what do you think ?